toolchain/parse/handle_expr.cpp

```diff
@@ -279,44 +280,72 @@ auto HandleExprLoop(Context& context) -> void {
         break;
 
       default:
-        state.state = State::ExprLoopForBinary;
+        state.state = State::ExprLoopForInfixOperator;
         break;
     }
 
     context.PushState(state);
     context.PushStateForExpr(operator_precedence);
   } else {
-    context.AddNode(NodeKind::PostfixOperator, state.token, state.subtree_start,
-                    state.has_error);
+    context.AddNode(NodeKind::PostfixOperatorStar, state.token,
```

Comment:
Even if we only have one postfix operator for now, I think we should add a `switch` here and a `..._POSTFIX` macro.

Reply:
Okay, I'll send a separate PR for that.